hkmc2/shared/src/test/mlscript-compile/FingerTreeList.mls

```diff
@@ -17,21 +17,19 @@ fun normIdxSlice(i, len) =
     else 0
   else i
 
-// Node[T]
-class Branch2(val size, val e1, val e2)
-class Branch3(val size, val e1, val e2, val e3)
+type Node[T] = Branch2[T] | Branch3[T]
```

Comment:
Do these annotations look ok?

Reply:
Sure, they look ok to me, but I don't know if they're correct.